test/samples/simple/simple.rs

```diff
@@ -1,22 +1,27 @@
-use std::io;
-
 /// * `n` - the first number
 /// * `other_number` - the second number
 fn simple(n: i32, other_number: i32) {
     /* TODO Just do what you want with these numbers, like sum them. */
 }
 
 fn main() {
-    let n: i32 = read_line().parse().unwrap();
-    let other_number: i32 = read_line().parse().unwrap();
+    let mut buffer = String::new();
+
+    let n = read_line(&mut buffer)
```

Comment:
Even though the norm is usually to avoid type annotation whenether you can, I kind of agree with you that it can make it harder to read in this case.

What actually makes me think it can be better in this case is that the candidate can change the type of an input variable by just changing it in the function parameter, at exactly one place. This can be very useful as the strong type system of Rust won't let you use an i32 when another integer type is expected (such as `usize` for indexing arrays).

Reply:
Ok, it is not a big deal if the main function is a bit hard to read, so we can keep it like you did.